server/models/User.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const UserSchema = new Schema({
```

Comment:
GitHub username? 
Being used quite a bit

Reply:
I was thinking name could be that